cleanlab/filter.py

```diff
@@ -59,6 +59,21 @@
     w = """To see estimated completion times for methods in cleanlab.filter, "pip install tqdm"."""
     warnings.warn(w)
 
+# psutil used to count physical cores for multiprocessing
```

Comment:
Thought you would add the change to count physical cores in separate PR, but happy to just take it from here in this one!

A few changes I'd request:

- [ ] Delete the warning here if `not psutil_exists`. I know there is an analogous warning for tqdm, but that's actually one of our dependencies, so that code is actually  unnecessary now.
- [ ] instead move this warning to be a print statement that occurs after the `if psutil_exists:` nested block inside of `find_label_issues()`.
- [ ] The message can read: """To default to the number of physical cores for multiprocessing in `find_label_issues()`, please: "pip install psutil"."""`
- [ ] Add psutil to list of dependencies in: https://github.com/cleanlab/cleanlab/blob/master/requirements-dev.txt

Reply:
thanks for fixing these!